examples/hybrid_jobs/7_Running_notebooks_as_jobs/Running_notebooks_as_jobs.ipynb

```diff
@@ -0,0 +1,439 @@
+{
```

Comment:
<p>Where is the cost metric defined? I don't see it anywhere in the script, and running this cell throws a <code>KeyError</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
The jobs metrics, `metrics = job.metrics()` fetches the metrics for the job which is in this case;
```
{'timestamp': [1681912863.4931602], 'cost': [0.00375], 'counts_00': [488.0]}
```
So `cost = metrics["cost"][0]=0.00375`.